DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/Misc.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef __MISC_HPP__

```

Comment:
Misc is a bad name for this file.  How about 'static_math'?

Reply:
Ran clang-format, renamed Misc as you suggested.